src/number_theory/bernoulli_polynomials.lean

```diff
@@ -123,23 +122,23 @@ begin
 end
 
 open power_series
-open polynomial (aeval)
 variables {A : Type*} [comm_ring A] [algebra ℚ A]
 
 -- TODO: define exponential generating functions, and use them here
 -- This name should probably be updated afterwards
 
 /-- The theorem that `∑ Bₙ(t)X^n/n!)(e^X-1)=Xe^{tX}`  -/
-theorem exp_bernoulli_poly' (t : A) :
-  mk (λ n, aeval t ((1 / n! : ℚ) • bernoulli_poly n)) * (exp A - 1) = X * rescale t (exp A) :=
+theorem exp_bernoulli' (t : A) :
```

Comment:
You can `end polynomial` before this lemma, since it is about power series and not about polynomials. Then we should find a better name for it.

I know that it can be very frustrating to being asked to modify a lot of names, but the name convention is really really useful when one gets used to it.

Reply:
Hmm, no, ok, it's ok in the polynomial namespace.

Reply:
Should we try adding it to the `power_series` namespace ?